core/src/main/scala/kafka/admin/AclCommand.scala

```diff
@@ -292,7 +293,7 @@ object AclCommand extends Logging {
     if (opts.options.has(opts.forceOpt))
         return true
     println(msg)
-    Console.readLine().equalsIgnoreCase("y")
```

Comment:
Can we also fix this in TopicCommand.scala?

Reply:
True, we can also fix it even though in `TopicCommand` it's currently dead code as `askToProceed()` is not called.